### PR TITLE
Extension build process

### PR DESCRIPTION
**Pull request by bradley-erickson**

### CODE REVIEW

**Comment by bradley-erickson** [comment]
@DrLynch Have you gotten a chance to review this process yet? We ought to get this merged in.

File: extension/writing-process/package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+    "name": "writing-process",
+    "description": "Tracks writing in Google Docs, and provides nifty insights to you and your teachers!",
+    "version": "0.0.1",
+    "homepage": "http://chrome.google.com/webstore",
+    "author": "ENTER YOUR NAME HERE",
```
Comment (pmitros) [inline extension/writing-process/package.json:6]: Enter our names here.

File: extension/writing-process/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "repository": {
+        "type": "git",
+        "url": "ENTER GIT REPO URL"
```
Comment (pmitros) [inline extension/writing-process/package.json:9]: Enter https://github.com/ETS-Next-Gen/writing_observer here (or even the path to the extension)

**Comment by pmitros** [comment]
As per our conversation, I would be inclined to merge this sooner rather than later (e.g. when my two comments are addressed), and then keep working from there. We do also need to check for outstanding issues on the NCSU side, to avoid merge conflicts. Once the two changes are made, and we're clear with NCSU, I approve merge.

**Comment by bradley-erickson** [comment]
@DrLynch we decided to merge this at it was a blocking step for Piotr. We can continue to work on beyond this.

**Pull request merged**